### PR TITLE
Use travis' support of yarn

### PR DESCRIPTION
**Pull request by samuelli**
Source: https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,10 +26,8 @@ env:
 before_install:
 - mkdir -p lib
 install:
-- npm install -g yarn@0.16
-- npm install -g bower
-- npm install -g web-component-tester
 - yarn
+- npm install -g bower
```
Comment (abdonrd) [inline .travis.yml]: Wouldn't it be better to use `yarn global add bower`?

File: .travis.yml [hunk 2]
```diff
@@ -38,10 +36,11 @@ install:
   && printf '\ny\n\ny\ny\n' | ./google-cloud-sdk/install.sh && cd $TRAVIS_BUILD_DIR;
   fi
 - printf '\ny\n' | $HOME/gcloud/google-cloud-sdk/bin/gcloud components install app-engine-python
+- alias wct="$TRAVIS_BUILD_DIR/client/node_modules/web-component-tester/bin/wct"
 script:
 - grunt lint
 - python tests.py $HOME/gcloud/google-cloud-sdk
-- xvfb-run wct --skip-plugin sauce
+- xvfb-run $TRAVIS_BUILD_DIR/client/node_modules/web-component-tester/bin/wct --skip-plugin sauce
 - "(cd analysis; npm test)"
```
Comment (abdonrd) [inline .travis.yml]: Wouldn't it be better to use `"(cd analysis; yarn test)"`?

File: .travis.yml (hunk 1 above)
Comment (samuelli) [inline .travis.yml]: I'll try it!

File: .travis.yml (hunk 2 above)
Comment (samuelli) [inline .travis.yml]: I don't see any benefit of yarn as a task runner

Comment (abdonrd) [inline .travis.yml]: I only suggest it to use always the same command.

Comment (samuelli) [inline .travis.yml]: I see. I think people are familiar with the existing one, so I'll leave it for now.

`yarn` build is significantly faster and is deterministic, so its worth the switch here.

**Pull request merged**